importer-config.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
 # How to find your accountId: https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/account-id/
-accountId=
+accountId=10956800
 # How to create a User API Key: https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#user-key-create
-apiKey=
+apiKey=NRII-5DYAYH7YpiNVIThYHXRz75z1Ld8ls_9N
```

Comment:
@rudouglas You're gonna want to rotate this API key 😄

Reply:
says you 😂